onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp

```diff
@@ -615,13 +615,30 @@ namespace Dml
 
     bool IsCpuOnDmlOperator(const onnxruntime::Node& node)
     {
-        auto sequence_ops = std::array<char*, 6>{
+        auto sequence_ops = std::array<char*, 7>{
             "SequenceAt",
             "SequenceConstruct",
             "SequenceEmpty",
             "SequenceLength",
             "SequenceErase",
             "SequenceInsert",
+            "Loop"
+        };
+
+        for (auto& sequence_op : sequence_ops)
+        {
+            if (strcmp(sequence_op, node.OpType().c_str()) == 0)
```

Comment:
[minor] If you use array of `string_view` instead of raw `char`, then you can just say sequence_op == node.OpType(). (easier to understand than strcmp's mental inversion to realize that 0 actually means true)

Reply:
Nice to follow internal consistency of coding style within a file `sequenceOps ` rather than `sequence🐍ops` to follow internal consistency within a file (file > module > project), but I sign off either way. Looks like a nicely targeted small change.